sdk/contentsafety/azure-ai-contentsafety/README.md

```diff
@@ -0,0 +1,272 @@
+# Azure ContentSafety client library for Java
+
+[Azure AI Content Safety][contentsafety_overview] detects harmful user-generated and AI-generated content in applications and services. Content Safety includes several APIs that allow you to detect material that is harmful:
+
+* Text Analysis API: Scans text for sexual content, violence, hate, and self harm with multi-severity levels.
```

Comment:
It is mentioned that "Content Safety includes several APIs that allow you to detect material that is harmful" but only 3 APIs are mentioned in the list. Make sure all the API descriptions are included in the list.

Reply:
we describe all blocklist APIs in Text Blocklist Management APIs
It is aligned with other three languages.